src/components/postCard.js

```
@@ -0,0 +1,129 @@
+`
+
+const StyledLink = styled(Link)`
+  color: var(--textNormal);
+  text-decoration: none;
+  
+  &:hover {
+    color: var(--textLink);
+  }
+`
+
+const Date = styled.small`
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)